common/autoware_universe_utils/src/geometry/ear_clipping.cpp

```diff
@@ -561,6 +564,28 @@ void ear_clipping_linked(
   }
 }
 
+double calculate_triangle_area(const autoware::universe_utils::Polygon2d & triangle)
+{
+  const auto & points = triangle.outer();
+  double x1 = points[0].x();
+  double y1 = points[0].y();
+  double x2 = points[1].x();
+  double y2 = points[1].y();
+  double x3 = points[2].x();
+  double y3 = points[2].y();
+
+  return std::abs((x1 * (y2 - y3) + x2 * (y3 - y1) + x3 * (y1 - y2)) / 2.0);
```

Comment:
Can you add a source for this formula ?
Or if this function is only needed for the test then just use `boost::geometry::area` in the tests.